CHANGES

```diff
@@ -1,3 +1,7 @@
+0.188
+
+- Remove status message support from HTTP client.
```

Comment:
What's the motivation?

Reply:
It isn't really supported anymore in the Jetty clients, and for HTTP/2 you just get `null`.  Also we never use it.

Reply:
Ok, maybe we should add that to the commit message